src/Sulu/Component/Webspace/Url.php

```diff
@@ -245,8 +245,8 @@ public function setEnvironment($environment)
      */
     public function isValidLocale($language, $country)
     {
-        return (empty($this->getLanguage()) || $this->getLanguage() === $language)
-            && (empty($this->getCountry()) || $this->getCountry() === $country);
+        return ($this->getLanguage() === $language && $this->getCountry() === $country)
+            || (empty($this->getLanguage()) && empty($this->getCountry()));
```

Comment:
What is this for needed now? The URL should not match if just both values are empty. And it should never be the case that both values are empty in this object.